### DISCUSSION TITLE
PowerShell Crescendo json files should support multiple native command definitions

### DISCUSSION THREAD

**Discussion by theJasonHelmick** [PowerShell/Crescendo#45] [Ideas]
PowerShell Crescendo currently supports generating multiple json files into a single PowerShell module.

```powershell
Export-CrescendoModule -ConfigurationFile .\get-ipconfig.proxy.json,.\Remove-ipconfig.proxy.json,.\New-ipconfig.proxy.json -ModuleName ./ipconfig.psm1
```

- It would be easier to develop and maintain multiple native commands in a single json file. 
- Crescendo should support both linux and windows native commands in the same json file.

**Comment by gaelcolas** [comment]
Yes, that's a must have IMO.

**Comment by P0w3rChi3f** [comment]
I like it.  It sounds like it would work perfectly for a tool I have in mind.  I think it would make updating the Crescendo module easier if you only had to update/modify one .json file.

**Comment by jdhitsolutions** [comment]
I would like to take this a step further and have an option for the export step to create a separate ps1 file for each function, instead of dumping them into a single psm1 file. Give me the option to specify a directory for the function files and have them dot sources in the psm1 file and exported in a module manifest.
